packages/web-components/fast-foundation/src/tree-item/tree-item.ts

```diff
@@ -55,8 +56,9 @@ export class FASTTreeItem extends FASTElement {
      */
     @attr({ mode: "boolean" })
     public expanded: boolean = false;
-    protected expandedChanged(): void {
+    protected expandedChanged(prev: boolean | undefined, next: boolean): void {
         if (this.$fastController.isConnected) {
+            this.ariaExpanded = next && this.childItems.length ? "true" : "false";
```

Comment:
This likely needs to change given the comment above about _not_ setting aria-expanded when there are no child items.